### PR TITLE
Add gFlex recipe

### PR DESCRIPTION
**Pull request by rbeucher**
Hi @conda-forge/help-python, 

Please consider adding this package to conda-forge. It is rather broadly used in the scientific community.
It is also an important dependency for packages that I would like to submit.

Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [X] Source is from official source
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/gFlex/LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,674 @@
...
+
+  You should also get your employer (if you work as a programmer) or school,
+if any, to sign a "copyright disclaimer" for the program, if necessary.
+For more information on this, and how to apply and follow the GNU GPL, see
+<http://www.gnu.org/licenses/>.
+
+  The GNU General Public License does not permit incorporating your program
+into proprietary programs.  If your program is a subroutine library, you
+may consider it more useful to permit linking proprietary applications with
+the library.  If this is what you want to do, use the GNU Lesser General
```
Comment (marcelotrevisani) [inline recipes/gFlex/LICENSE.md:672]: it seems the license is already being packaged, you can delete this file please

File: recipes/gFlex/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+
+about:
+  home:
+    - https://github.com/awickert/gFlex
+  license: GPL-3.0-only
+  license_family: GPL3
+  license_file: LICENSE.md
+  summary: "One- and two-dimensional plate bending, designed for Earth's lithosphere"
+  doc_url: 
+  dev_url:
```
Comment (marcelotrevisani) [inline recipes/gFlex/meta.yaml:42]: Please delete the empty sections
```suggestion
```

File: recipes/gFlex/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - gflex
+  commands:
+    - gflex --help
+
+about:
+  home:
+    - https://github.com/awickert/gFlex
```
Comment (marcelotrevisani) [inline recipes/gFlex/meta.yaml:36]:
```suggestion
  home: https://github.com/awickert/gFlex
```

File: recipes/gFlex/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,46 @@
+{% set name = "gflex" %}
+{% set version = "1.1.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://github.com/awickert/gFlex/archive/v1.1.0.tar.gz
```
Comment (marcelotrevisani) [inline recipes/gFlex/meta.yaml:9]:
```suggestion
  url: https://github.com/awickert/gFlex/archive/v{{ version }}.tar.gz
```

File: recipes/gFlex/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
...
+  sha256: dc722606dbc6a549c84bc86849419bc43e4fcd9c0d8329984fe6fbf6b0b209d4
+
+
+build:
+  number: 0
+  entry_points:
+    - gflex = gflex:main
+  script: "{{ PYTHON }} -m pip install . -vv"
```
Comment (marcelotrevisani) [inline recipes/gFlex/meta.yaml:17]: I believe this recipe can be ``noarch``, right? could you please change it to:
```suggestion
  script: "{{ PYTHON }} -m pip install . -vv"
  noarch: python
```

File: recipes/gFlex/meta.yaml [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+  run:
+    - python
+    - scipy
+    - matplotlib
+
+test:
+  imports:
+    - gflex
+  commands:
+    - gflex --help
```
Comment (marcelotrevisani) [inline recipes/gFlex/meta.yaml:32]: Can we try to add ``pip check`` here please?
```suggestion
    - gflex --help
    - pip check
  requires:
    - pip
```

**Comment by rbeucher** [comment]
Thanks @marcelotrevisani ! Great suggestions

**Review by marcelotrevisani — APPROVED** [review]

**Comment by rbeucher** [comment]
The feedstock is failing. not sure what is going on there...

**Pull request merged**